node.lua

```diff
@@ -71,6 +73,26 @@ categories["u_bahn"] = resource.load_image("u_bahn.png")
 categories["tram"] = resource.load_image("tram.png")
 categories["bus"] = resource.load_image("bus.png")
 
+nre_powered_logo = resource.load_image{
```

Comment:
This should only be loaded if it should actually be shown. Currently it's fine, but some day we maybe have lots of operator logos, so we should only load what's actually needed.

Reply:
Done.